### PR TITLE
postgres_and_docker

### PR DESCRIPTION
**Pull request by Rivallar**
Добавил postgres и docker

### CODE REVIEW

File: autosaloon/settings.py [hunk 1]
```diff
@@ -20,7 +22,7 @@
 # See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-nv%9k#xpi^z6dt#7rlrt5$w%epq7n6w%nwd5g*#4$nbg0a4upm'
+SECRET_KEY = KEY
```
Comment (achopik) [inline autosaloon/settings.py]: Use os.getenv instead and inject all params throug env variables

File: autosaloon/setup_and_security.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+KEY = 'django-insecure-nv%9k#xpi^z6dt#7rlrt5$w%epq7n6w%nwd5g*#4$nbg0a4upm'
```
Comment (achopik) [inline autosaloon/setup_and_security.py]: Do not store keys in code, inject them using env. Secret key leak is a high-score security vulnerability so avoid it at all cost

File: autosaloon/setup_and_security.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+POSTGRES_DB = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'Autosaloon',
+        'PASSWORD': 'AutosaloonBOSS',
+        'HOST': 'db',
+        'PORT': 5432,
+        }
+    }
```
Comment (achopik) [inline autosaloon/setup_and_security.py]: Same about DB credentials and configs. Pass them using env files

**Review by achopik — CHANGES_REQUESTED** [review]
Some changes requested

File: autosaloon/setup_and_security.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': os.getenv('DB_USER'),
+        'PASSWORD': os.getenv('DB_PASS'),
+        'HOST': os.getenv('DB_HOST'),
+        'PORT': os.getenv('DB_PORT'),
+        }
+    }
+
```
Comment (achopik) [inline autosaloon/setup_and_security.py]: For now, you don't need to move it into another module, keep in in settings file

**Review by achopik — CHANGES_REQUESTED** [review]
One change left

File: autosaloon/setup_and_security.py (hunk 4 above)
Comment (Rivallar) [reply autosaloon/setup_and_security.py]: Done. Also added to .env database NAME as I have different db names on my PC and laptop.

**Pull request merged**